telethon_examples/README.md

```diff
@@ -136,6 +136,7 @@ assumes some [`asyncio`] knowledge, but otherwise is easy to follow.
 * Difficulty: **medium**.
 
 This example shows how to make invoices (Telegram's way of requesting payments) via a bot account. The example does not include how to add shipping information, though.
+You'll need to obtain provider token, more info [here](https://core.telegram.org/bots/payments)
```

Comment:
Please avoid using links which are just a single word like "[here](https://example.com)". They provide no context whatsoever on their own and are annoying accessibility-wise.
```suggestion

You'll need to obtain a "provider token" to use this example, so please read [Telegram's guide on payments](https://core.telegram.org/bots/payments) before using this example.

```

(The empty newline is necessary or Markdown won't make it separate paragraphs.)